qiskit/circuit/quantumcircuit.py

```diff
@@ -2265,17 +2265,98 @@ def reset(self, qubit: QubitSpecifier) -> InstructionSet:
         return self.append(Reset(), [qubit], [])
 
     def measure(self, qubit: QubitSpecifier, cbit: ClbitSpecifier) -> InstructionSet:
-        """Measure quantum bit into classical bit (tuples).
+        """Measure quantum bit (qubit) into classical bit (cbit).
+
+        When a qubit is measured, its state collapses to a classical bit and copied to a
+        classical wire.
 
         Args:
-            qubit: qubit to measure.
-            cbit: classical bit to place the measurement in.
+            qubit: qubit/s to measure.
+            cbit: classical bit/s to place the measurement in.
```

Comment:
```suggestion
            qubit: qubit(s) to measure.
            cbit: classical bit(s) to place the measurement result(s) in.
```

Neither is particularly standard, but using "(s)" to denote "maybe plural" is probably the most common way to do it in English.